### DISCUSSION TITLE
Ignore PR Limit For Rule

### DISCUSSION THREAD

**Discussion by beeequeue** [renovatebot/renovate#10568] [Suggest an Idea]
Is there a way to ignore the PR limit in a `packageRules` entry? e.g. if we wanted PRs for our private scope to always be created, even if there are enough PRs open currently. 

```json
{
  "matchPackagePrefix": "@scope/",
  "ignorePrLimit": true
}
```

**Comment by viceice** [comment]
nope

**Reply by beeequeue** [reply]
Re-categorized the discussion to ideas instead
